src/fmu/dataio/dataio.py

```diff
@@ -796,13 +796,13 @@ def export(
 #
 # The InitializeCase is used for making the case matadata prior to any other actions,
 # e.g. forward jobs. However, case metadata file may already exist, and in that case
-# this class should only emit a message or warning
+# this class should only emit a message or warning.
 # ######################################################################################
 
 
 @dataclass
 class InitializeCase:  # pylint: disable=too-few-public-methods
-    """Instantate InitializeCase object.
+    """Initialize metadata for an FMU Case.
```

Comment:
Not relevant for this PR, but part of bigger discussion: In a big renaming effort a while back, we renamed everything from "initialize case" to "create case metadata" - except the actual class name inside fmu-dataio.

Reply:
https://github.com/equinor/fmu-dataio/issues/171